packages/core/test/oracle/VotingV2.js

```diff
@@ -2663,6 +2663,93 @@ describe("VotingV2", function () {
     await voting.methods.setDelegator(ZERO_ADDRESS).send({ from: rand });
     assert.equal(await voting.methods.delegateToStaker(rand).call(), ZERO_ADDRESS);
   });
+
+  it("Emit events", async function () {
```

Comment:
this feels somewhat similar to the events here: https://github.com/UMAprotocol/protocol/blob/94d53ffc7fdafee7b5a4443d83dd50d02c0e72e2/packages/core/test/oracle/VotingV2.js#L887. do you want to shorten this to _just_ be `Emit StakeEvents` or perhaps merge this with the tests there? Also, for the staking specific tests it might be better to pull some of this into this location: https://github.com/UMAprotocol/protocol/blob/master/packages/core/test/oracle/Staker.ts

sorry that I'm making you move stuff around...Just would ideally not like too much duplication in the tests.

Reply:
No problem! I missed that file and existing test! I have reorganised the code reuse the existing tests and moved the staking ones to the corresponding file.